### DISCUSSION TITLE
Turkish language support

### DISCUSSION THREAD

**Discussion by lim10soft** [Blair2004/NexoPOS#745] [Ideas]
Hello, I am serving in Turkey, since there is no Turkish language support in the program, I have difficulties in continuous updates. If it is suitable for you, can you bring Turkish language support. Thank you.

**Comment by Blair2004** [comment]
Hi, 
approved. We hope you'll help making the Turkish better.

**Reply by lim10soft** [reply]
Of course, sir, I will translate it into Turkish and present it to you.

**Reply by lim10soft** [reply]
I translated the latest version of nexopos into Turkish for you.
https://dosya.co/a41l3nl949cw/tr.json.html

**Comment by lim10soft** [comment]
File Edited.
NEXOPOS TR-
https://dosya.co/a41l3nl949cw/tr.json.html
GASTROPOS TR-
https://dosya.co/8l07pcf3kkha/tr.json.html

NPS PRİNT ADAPTÖR TR-
https://dosya.co/esr31i6xsfea/tr.json.html

**Reply by Blair2004** [reply]
Thanks, I'll be added from the next release.
